app/main/forms/direct_award_forms.py

```diff
@@ -39,3 +44,138 @@ class DidYouAwardAContractForm(FlaskForm):
             (STILL_ASSESSING, 'We are still assessing services')
         ]
     )
+
+
+class DMFormField(FormField):
+    @property
+    def errors(self):
+        return [e for errors in self.form.errors.values() for e in errors]
```

Comment:
Unless this needs to return a list, then generator-style (i.e. with regular not square brackets) would probably be better.

If this thing, and the below validators, feel like they aren't specifically tied to the Direct Award flow, then let's a least put them into a separate module, even if we aren't yet ready to move them into dm-utils.

Reply:
It needs to be a list as the property it is overloading returns a list that is usually accessed using `__getitem__`.

I will move them into a separate module.